modules/learn/pages/data/transactions.adoc

```diff
@@ -45,44 +30,78 @@ If two transactions simultaneously attempt to write to the same document, one is
+Note that data within a single document is _always_ updated atomically (without the need for transactions): therefore, whenever multiple key-value pairs consistently require atomic update, their co-location within a single document may best ensure high performance.
 
-* xref:learn:clusters-and-availability/xdcr-overview.adoc[Cross Data Center Replication (XDCR)] supports _eventual consistency_: however, it does _not_ support atomicity &#8212; nor does
+* xref:learn:clusters-and-availability/xdcr-overview.adoc[Cross Data Center Replication (XDCR)] supports _eventual consistency_: however, it does not support atomicity -- nor does
 xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc[XDCR Conflict Resolution].
 Consequently, transactionally modified documents should only be replicated across clusters if no transactions involving the same documents can occur on those clusters simultaneously.
+In other words, Active-Active XDCR deployment (that is, where both clusters are being written to simultaneously) is not supported with the use of transactions and can lead to unexpected behavior and data loss.
 +
 See xref:learn:clusters-and-availability/xdcr-overview.adoc#xdcr-direction-and-topology[XDCR Direction and Topology], for a description of _bidirectional_ XDCR.
 See xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc#ensuring_safe_failover[Ensuring Safe Failover], for information on failing a transactional application from one data center to another.
 
 * Since transactions make use of the xref:learn:data/durability.adoc[Durability] provided by _synchronous writes_, in order to use transactions in development on a _single-node cluster_, the number of replicas assigned to any bucket to be used transactionally must be established as _zero_: if the number is greater than zero, the required durability level is unattainable, and the write fails.
 +
-If, on a single-node cluster, a bucket's replica-assignment was previously greater than zero, and the number is reduced to zero in order to support synchronous writes, a rebalance must be performed prior to any synchronous write.
+NOTE: If using a single node cluster (for example, during development), then note that the default number of replicas for a newly created bucket is 1.
+If left at this default, then all Key-Value writes performed at with durabiltiy will fail with a `DurabilityImpossibleException`.
+In turn this will cause all transactions (which perform all Key-Value writes durably) to fail.
+This setting can be changed via xref:manage:manage-buckets/create-bucket.adoc#couchbase-bucket-settings[GUI] or xref:cli:cbcli/couchbase-cli-bucket-create.adoc#options[command line].
+
+* A Transactional N1QL Query using `AT_PLUS` will be automatically upgraded to `REQUEST_PLUS`.
```

Comment:
I don't know.  This is already a somewhat scary list of limitations and gotchas, I don't know how much we want to put little details like this front and centre.